tests/testthat/indention_token_dependent/token_dependent_comments-out.R

```diff
@@ -0,0 +1,7 @@
+call(call( # comment
+  3, 4
+))
+
+call(call(1, # comment
+          3
```

Comment:
We could indent with two space if there's a break, or optionally indent below the opening paren. Adding line breaks looks like an independent problem to me.